UI/ObjectListWnd.cpp

```diff
@@ -1042,7 +1035,7 @@ void FilterDialog::CompleteConstruction() {
     m_filters_layout->SetColumnStretch(0, 0.0);
 
     GG::X button_width = GG::X(ClientUI::Pts()*8);
-    GG::Button* label = nullptr;
+    std::shared_ptr<GG::Button> label = nullptr;
```

Comment:
Is `shared_ptr` something different after default initialization than the result of assigning it `nullptr` ?